src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1180,13 +1180,13 @@
     <value>NTLM authentication is not possible with default credentials on this platform.</value>
   </data>
   <data name="net_nego_not_supported_empty_target_with_defaultcreds" xml:space="preserve">
-    <value>Target name should be non-empty if default credentials are passed.</value>
+    <value>Target name should be non empty if default credentials are passed.</value>
   </data>
   <data name="net_nego_server_not_supported" xml:space="preserve">
-    <value>Server implementation is not supported.</value>
+    <value>Server implementation is not supported</value>
   </data>
   <data name="net_nego_protection_level_not_supported" xml:space="preserve">
-    <value>Requested protection level is not supported with the GSSAPI implementation currently installed.</value>
+    <value>Requested protection level is not supported with the gssapi implementation currently installed.</value>
```

Comment:
I believe GSSAPI should be capitalized as it's an acronym

Reply:
done